deps/chakrashim/chakracore.gyp

```diff
@@ -7,11 +7,10 @@
     'msvs_windows_target_platform_version_prop': '',
     'icu_args%': '',
     'icu_include_path%': '',
+    'icu_lib_path%': '',
     'linker_start_group%': '',
     'linker_end_group%': '',
     'chakra_libs_absolute%': '',
-    'icu_lib_path%': '',
-    'chakra_build_flags': '',
 
     # xplat (non-win32) only
     'chakra_config': 'Release',     # Debug, Release
```

Comment:
Sorry, meant to say - did you mean to change it to Debug? From your PR comment, it sounded like you were going to change the default too but guess not?